### DISCUSSION TITLE
What exactly was the thing that was broken that you've tried to fix?

### DISCUSSION THREAD

**Discussion by ChrisMaunder** [community/community#60725] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Why did you guys change the UI like this? Was there a huge pressing demand to make something we use to manage our critical code into Yet Another Social Network? 

It feels like the classic trap of taking something that works, and trying to make it do more, but in the process killing the very thing that makes it work. Instead of adding to the product, the product ends up diminished.

Sorry but I don't want trending repos. I don't want recommendations on things to follow. I want to be able to focus on ME and my team's code. I'm in the zone, I have 50 things going on, I head to GitHub to quickly check a repo so I can add a team member and also review a commit and... now I have to hunt around and wade through the distractions.

By all means make some changes. But do not forget the primary reason we're here: to manage **our** code. Make that front and centre. Make that perfect. Make that so quick and simple that we'll have time to wander around and check out other repo's.

**Comment by GhostOf0days** [comment]
I have no issues with the new UI. People are blowing it out of proportion. Learn and adapt. It's fine to criticize it, but phrasing it as if it's the worst thing ever isn't accurate.

**Reply by scarlet1nbythehill** [reply]
Github, you aren't social media. I don't need you to show me trendy repos, You are a work tool. It would be so much better if you showed me my repos instead of some trendy repo I don't even care about.
